src/clj/fluree/db/query/fql/parse.cljc

```diff
@@ -270,11 +270,15 @@
 
 (defn parse-where-clause
   [clause vars context]
-  (->> clause
-       util/sequential
-       (mapcat (fn [pattern]
-                 (parse-pattern pattern vars context)))
-       where/->where-clause))
+  ;; a single non-node where pattern is already sequential, so we need to specially handle it
+  (let [clause* (if (and (sequential? clause) (keyword? (first clause)))
```

Comment:
This condition should be factored into a predicate function with a name like `higher-order-pattern?`